src/main/java/reposense/util/FileUtil.java

```diff
@@ -2,33 +2,41 @@
 
 import java.io.File;
 import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.OutputStream;
 import java.io.PrintWriter;
+import java.nio.ByteBuffer;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.util.ArrayList;
 import java.util.Comparator;
+import java.util.List;
 import java.util.logging.Level;
 import java.util.logging.Logger;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 import java.util.zip.ZipEntry;
 import java.util.zip.ZipInputStream;
+import java.util.zip.ZipOutputStream;
 
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
 
-import reposense.RepoSense;
 import reposense.system.LogsManager;
 
 
 public class FileUtil {
     private static Logger logger = LogsManager.getLogger(FileUtil.class);
-
     private static final String GITHUB_API_DATE_FORMAT = "yyyy-MM-dd";
 
+    // zip file which contains all the specified file types
+    private static final String ZIP_FILE = "archive.zip";
```

Comment:
make this `public` and import it for use in your unit test instead of redefining